packages/klighd-core/src/sidebar/counter-panel.tsx

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
If we don't use this in the package directly I think code like this should be in some seperate examples folder to keep the package itself clean.

Reply:
Moved to `packages/klighd-core/examples` which is not build and packaged when distributed.